README.md

```diff
@@ -1,205 +1,27 @@
-# Vault Plugins: MongoDB Atlas Secrets Engine and Database Secrets Engine for MongoDB Atlas plugin
+# HashiCorp Vault Database Secrets Engine - MongoDB Atlas plugin
 
-**IMPORTANT: This plugin is currently under development.  Feel free to test it out following the instructions under the Developing section below, however consider this beta until it is verified by HashiCorp. Once verified and released versions will be documented in a CHANGELOG**
+MongoDB Atlas is one of the supported plugins for the HashiCorp Vault Database Secrets Engine and allows for the programmatic generation of unique, ephemeral MongoDB [Database User](https://docs.atlas.mongodb.com/reference/api/database-users/) credentials in MongoDB Atlas Projects.
 
-This contains two Secrets Engines specific to MongoDB Atlas for use with [Hashicorp Vault](https://github.com/hashicorp/vault).
-The first is the MongoDB Atlas Secrets Engine which generates unique, ephemeral [Programmatic API](https://docs.atlas.mongodb.com/reference/api/apiKeys/) keys for MongoDB Atlas.
-The second is an extension of the existing Database Secrets Engine and allows generation of unique, ephemeral
-programmatic MongoDB [Database User](https://docs.atlas.mongodb.com/reference/api/database-users/) credentials in MongoDB Atlas, thus we refer to it as the Database Secrets
-Engine for MongoDB Atlas.
+**The plugin is included in version 1.4 of Vault.**
 
-The plugins are located in the following directories:
-  - **MongoDB Atlas Secrets Engine:** `plugins/logical/mongodbatlas/`
-  - **Database Secrets Engine for MongoDB Atlas plugin:** `plugins/database/mongodbatlas`
+This repository will be archived in the near future as the code developed here is now hosted within HashiCorp's Vault repo: https://github.com/hashicorp/vault.  Any PRs or issues should be filed there.  Please see Vault's README for more information.
```

Comment:
As I understood from one of our conversations with @kalafut we will maintain and then sync with the Vault core, so we will maintain using this repo right? Can you confirm @kalafut?

Reply:
The plan changed a couple of times, but we've landed on the hashicorp repos being the canonical source, in particular since the plugin is bundled with Vault. I prompted @themantissa about this recently, suggesting that having both sets of repos present/active would be confusing for issue or PR submitters. Provided you've subscribed to the Hashicorp ones, you'll be notified of activity there. And we'll definitely be reaching out for anything beyond very minor issues or PRs that come in.

Reply:
Alright, awesome